SampleApps/SenderDemo/ActionData/DPLActionDataSource.m

```diff
@@ -74,4 +75,14 @@ - (DPLDemoAction *)logHelloWorldAction {
     return action;
 }
 
+
+- (DPLDemoAction *)logShoppingList {
+    DPLMutableDeepLink *link = [[DPLMutableDeepLink alloc] initWithString:@"dpl://shoppinglist?list%5B%5D%3DBread&list%5B%5D%3DCandies&list%5B%5D%3DWine&list%5B%5D%3DBeer"];
```

Comment:
I was wondering if here should be easier way to initialize deep link with URLs of such kind?